src/charm.py

```diff
@@ -761,10 +760,10 @@ def request_certificates(self):
             "kubernetes.default",
             "kubernetes.default.svc",
             f"kubernetes.default.svc.{domain}",
+            *node_address.by_relation(self, "kube-control", True),
         ]
         sans += bind_addrs
         sans += config_addrs
-        sans += ingress_addrs
```

Comment:
replace with the addresses from `node_address.by_relation(self, "kube-control", True)`